terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-up.ts

```diff
@@ -60,7 +60,7 @@ export async function scaleUp(
   const runnerTypes = await getRunnerTypes(
     {
       owner: repo.owner,
-      repo: Config.Instance.enableOrganizationRunners ? Config.Instance.scaleConfigRepo : repo.repo,
+      repo:  Config.Instance.scaleConfigRepo || repo.repo,
```

Comment:
Inconsistent behaviour with terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-up.ts 

I like the other one best (using the organization when defined) so maybe we can take the time to reflect the improvement here.

Reply:
After reading again, concluded that `ec2runner.org` should be `== to repo.owner` as it comes from that variable.

So there aren't inconsistencies